packages/scripts/promoteRelease.js

```diff
@@ -0,0 +1,242 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * @format
+ * @ts-check
```

Comment:
A description of what this script does / when I'd run it would be great to add to the comment header.

Reply:
Yeah, I can do that. Was also planning on adding a wiki page.